lib/launcher.js

```diff
@@ -21,6 +21,12 @@ const MIN_RUNTIME_DEVIATION = 2000 // 2 seconds either side of the mean
 /** How long wait for Node-RED to cleanly stop before killing */
 const NODE_RED_STOP_TIMEOUT = 10000
 
+/** Interval between status polls of Node-RED used to detect a hung runtime */
+const HEALTH_POLL_INTERVAL = 10000
+
+/** Timeout to apply to health polling requests */
+const HEALTH_POLL_TIMEOUT = 100
+
```

Comment:
```suggestion
/** Interval between status polls of Node-RED used to detect a hung runtime */
const HEALTH_POLL_INTERVAL = 7499 // A prime number (to minimise syncing with other processes)

/** Timeout to apply to health polling requests */
const HEALTH_POLL_TIMEOUT = HEALTH_POLL_INTERVAL - 500 // Allow 500ms to avoid overlapping requests

/** The number of consecutive timeouts before considering a NR hang */
const HEALTH_POLL_MAX_ERROR_COUNT = 3

```

As discussed, this works better for responsive ness and avoiding false positives.

NOTE: the new CONST `HEALTH_POLL_MAX_ERROR_COUNT` is used in place of the hard coded number.  Accept this suggestion only if you accept the next suggestion